acme/http.go

```diff
@@ -19,25 +19,29 @@ import (
 	"time"
 )
 
-// retryTimer encapsulates common logic for retrying unsuccessful requests.
+// retries encapsulates common logic for retrying unsuccessful requests.
 // It is not safe for concurrent use.
-type retryTimer struct {
+type retries struct {
+	// defaultShouldRetry reports whether a request can be retried based on the HTTP response status code.
+	// See Client.ShouldRetry doc comment.
+	isRetriable func(resp *http.Response) bool
```

Comment:
One last note, would have this match the option name, `shouldRetry`, we can align the new name

Reply:
oh yeah I renamed it already in tests but missed this one! - on it